### PR TITLE
French support

### PR DESCRIPTION
**Pull request by Morpheo1**

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<resources>
+    <string name="app_name">Polysmee</string>
+    <string name="appointment_reminder_notification_chanel_description">Channel de notification utilisé pour rappeler à l\'utilisateur qu\'il/elle a un meeting sous peu</string>
+    <string name="appointment_reminder_notification_notification_text">Rappel de meeting :)</string>
+    <string name="title_settings_appointments_reminder">Paramètres de rappel de meeting</string>
+    <string name="title_settings_appointments_reminder_notification_time_from_appointment">Me rappeler avant</string>
+    <string name="summary_settings_appointments_reminder_notification_time_from_appointment">Sélectionnez combien de temps avant un meeting vous devriez être prévenu (en minutes)</string>
+    <string name="title_settings_main_user_name">Nom d\'utilisateur</string>
+    <string name="title_settings_main_user_email">Email</string>
+    <string name="roomActivityReceiveButton">Recevoir</string>
```
Comment (tombenamato) [inline app/src/main/res/values-fr/strings.xml:11]: Why did you move them from settings.xml to here?

**Review by Fermeli — APPROVED** [review]
Good job

**Review by AttiaYoussef — APPROVED** [review]

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (Morpheo1) [reply app/src/main/res/values-fr/strings.xml:11]: This is the french translation, everything translated in french is in values-fr/strings.xml, regardless of their original file

Comment (tombenamato) [reply app/src/main/res/values-fr/strings.xml:11]: ok thanks

**Review by tombenamato — APPROVED** [review]
